### ISSUE TITLE
Self-sufficient villages

### ISSUE THREAD

**Issue by MaiyrCordeth** [TheResimulators/Sim-U-Kraft-Resimulated#7]
Maiyr: with the use of currency already. figured the villagers actions could be driven by money. like it was before no pay no mine. but with that pay they can buy their tools which of course pays the forger/blacksmither for the tools. the miner can sell his blocks to a store which sells the ores to the blacksmith the make the items
we could have different types of stores too, or just a "walmart" type of store

General5001: well there is already a builders merchant place but maybe we could make one for tools another one for ores

trying to make villagers self-sufficient

**Comment by fabbe50** [comment]
I really like this idea, it's what would differentiate this mod from other mods in the category. I'm gonna start planning out the mechanics for this, but I think it will take a while before features like this will make it into the finished mod.

**Comment by MaiyrCordeth** [comment]
yeah General and I thought about the fact we have currency, lets actually use it for something.

**Comment by general5001** [comment]
that sounds good fabbe I am sure some of these idea wont make it into a early development of the mod thanks for the response
